master.yaml

```diff
@@ -298,6 +321,12 @@ resources:
         - path: /usr/local/bin/retry
           permissions: 0755
           content: {get_file: fragments/retry.sh}
+        - path: /etc/sysconfig/network-scripts/ifcfg-eth1
+          content:
+            str_replace:
+              params:
+                $IFNAME: eth1
```

Comment:
I did so in the - very improbable - case where we would need to use it for an other interface. I can change it if you want